pageserver/src/consumption_metrics.rs

```diff
@@ -41,8 +41,8 @@ use reqwest::Url;
 /// ```
 #[serde_as]
 #[derive(Serialize, Deserialize, Debug, Clone, Eq, PartialEq, Ord, PartialOrd)]
-pub struct BillingMetric {
-    pub metric: BillingMetricKind,
+pub struct ConsumptionMetric {
+    pub metric: ConsumptionMetricKind,
```

Comment:
Can we rename fields here? Or this is already in some API contract? 

This is out of scope of this PR, but would be nice to do it earlier when we're not committed to backwards compatibility guarantees. Added some suggestions here: https://github.com/neondatabase/neon/pull/3215#discussion_r1058291562

Reply:
there is already existing spec
https://github.com/neondatabase/cloud/blob/a374c0e4c6b94116724137b186684453e785fe61/openapispec/billing/billing-v1.yaml#L133-L154

I don't think it is worth changing as long as there is a spec and everyone is agree on terms.
I'll add a `docs/consumption_metrics.md` that describes this.